respy/python/shared/fast_routines.py

```diff
@@ -0,0 +1,196 @@
+"""This module contains fast routines or replacements of numpy functions."""
```

Comment:
Do we really need all of those? Most of the time it's best to use numba only for the bottlenecks. I think it's dangerous to start re-implementing so many functions.

Reply:
You are right. This was unnecessary. Also, ``fr.clip`` was used once in a non-critical function. So, I deleted the complete module.